src/xdocs/checks/coding/covariantequals.xml

```diff
@@ -79,48 +79,40 @@ public boolean equals(Foo obj) {...}
           For example:
         </p>
         <source>
-class Test {
-  public boolean equals(Test i) {  // violation
+public class Example1 {
+  public boolean equals(Example1 same) {  // violation
     return false;
   }
+
+  record Test(String str) {
+    public boolean equals(Test r) {  // violation
+      return false;
+    }
+  }
 }
         </source>
         <p id="Example2-code">
           The same class without violations:
         </p>
         <source>
-class Test {
-  public boolean equals(Test i) {  // no violation
+public class Example2 {
+  public boolean equals(Example2 same) {  // no violation
     return false;
   }
 
-  public boolean equals(Object i) {
+  public boolean equals(Object o) {
     return false;
   }
-}
-        </source>
-        <p id="Example3-code">
-          Another example:
-        </p>
-        <source>
-record Test(String str) {
-    public boolean equals(Test r) {  // violation
-        return false;
-    }
-}
-        </source>
-        <p id="Example4-code">
-          The same record without violations:
-        </p>
-        <source>
-record Test(String str) {
+
+  record Test(String str) {
     public boolean equals(Test r) {  // no violation
-        return false;
```

Comment:
`equals(Test same)`

Please double check others like this.
"same" might help readers to treat it as "same type"
"o" is usual shortening of object.

Reply:
Done